wasm-crypto/src/lib.rs

```diff
@@ -475,6 +476,38 @@ pub fn encode_input_for_account_outpoint(
     Ok(input.encode())
 }
 
+#[wasm_bindgen]
+pub fn estimate_transaction_size(
+    inputs: &[u8],
+    mut opt_utxos: &[u8],
+    mut outputs: &[u8],
+) -> Result<usize, Error> {
+    let mut tx_outputs = vec![];
+    while !outputs.is_empty() {
+        let output = TxOutput::decode(&mut outputs).map_err(|_| Error::InvalidOutput)?;
+        tx_outputs.push(output);
+    }
+
+    let size = tx_size_with_outputs(&tx_outputs);
+    let inputs_size = inputs.len();
+
+    let mut total_size = size + inputs_size;
```

Comment:
Does inputs include the size of the input vec (= the number of inputs) at the beginning?